crates/libzkp/src/tasks.rs

```diff
@@ -79,29 +78,22 @@ pub fn gen_universal_chunk_task(
 /// Generate required staff for batch proving
 pub fn gen_universal_batch_task(
     task: BatchProvingTask,
-    _: ForkName,
 ) -> eyre::Result<(B256, BatchProofMetadata, ProvingTask)> {
-    let (batch_info, batch_hash) = task.precheck_and_build_metadata()?;
+    let (batch_info, batch_pi_hash) = task.precheck_and_build_metadata()?;
     let proving_task = task.try_into()?;
-
     Ok((
-        batch_hash,
-        BatchProofMetadata {
-            batch_info,
-            batch_hash,
-        },
+        batch_pi_hash,
```

Comment:
Oh, the previous `batch_hash` actually meant `batch_pi_hash`? That's confusing... Much clearer now.